05.wc/wc.rb

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options = ARGV.getopts('lwc')
+  filename = ARGV
+  option_values = { 'l' => 0, 'w' => 0, 'c' => 0 }
+
+  if !filename.empty?
+    file_read = filename.map { |file| File.read(file) }
+    words_width = file_read.map { |file_detail| file_detail.bytesize.to_s }.max.length
+    display_values(filename, file_read, options, words_width)
+    display_totals(option_values, file_read, options, words_width) if filename.length > 1
+  else
+    standard_input(options)
+  end
+end
+
+def standard_input(options)
+  words_width = 7
+  gets_file = readlines
+  gets_file = gets_file.join('')
```

Comment:
```suggestion
  gets_file = readlines.join
```

でも同じですね。

ただ、gets_fileだと「ファイルを取得する」というメソッドのように見えてしまいます